docs/UserGuide.md

```diff
@@ -196,19 +197,22 @@ MediBase3 can be controlled using commands in the **Command Line Interface (CLI)
 
 Each command follows this structure:
 
-![CommandPromptFormat.jpg](images%2FCommandPromptFormat.jpg)
+![CommandPromptFormat.jpg](images/CommandPromptFormat.jpg)
 
 Here’s a breakdown of the parts:
 
-| **Component** | **Description**                                    |
-|---------------|----------------------------------------------------|
-| **Command**   | The action you want to perform (e.g., `add`, `edit`) |
+| **Component** | **Description**                                         |
+|---------------|---------------------------------------------------------|
+| **Command**   | The action you want to perform (e.g., `add`, `edit`)    |
 | **Prefix**    | Identifies the data being entered (e.g., `n/` for name) |
 | **Parameter** | The value associated with the prefix (e.g., `John Doe`) |
 
 ### Example:
 
-add n/John Doe i/S1234567A d/2000-01-01 g/M p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25
+    ```
+    add n/John Doe i/S1234567A d/2000-01-01 g/M p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25
+    ```
```

Comment:
Might have to use single backticks for this, else it will result in a horizontal scrollbar. Our UG has to be PDF-friendly based on the cosntraints.

![image](https://github.com/user-attachments/assets/b6a3a62d-5132-482e-8643-493b3fdb42ad)

Reply:
Good point!

Reply:
![image](https://github.com/user-attachments/assets/210b6d49-646d-48e6-8dac-8d38ec2d1409)

regex is goated

Reply:
[MediBase3 User Guide.pdf](https://github.com/user-attachments/files/17497301/MediBase3.User.Guide.pdf)